auto-cpufreq-installer

```diff
@@ -78,7 +78,7 @@ function tool_install {
 
   if [ -f /etc/debian_version ]; then
```

Comment:
Since for anyone (or any distro derivative) that's using current (stable) version of Debian 12 or anything lower then Ubuntu 24.04 ([as I mentioned here](https://github.com/AdnanHodzic/auto-cpufreq/issues/813#issuecomment-2726246869)), let's extend this logic/loop so that it installs `libgirepository1.0-dev` in case of older releases otherwise go with `libgirepository-2.0-dev` for anything Debian >= 13 (trixie) and let's merge this.